### PR TITLE
Add preliminary support for florist friars

### PR DESCRIPTION
**Pull request by MrFizzyBubbs**

### CODE REVIEW

File: src/post.ts [hunk 1]
```diff
@@ -5,12 +5,20 @@ import {
...
 } from "kolmafia";
-import { $item, get, getRemainingStomach, property } from "libram";
+import { $item, $location, get, getRemainingStomach, property } from "libram";
+import {
+  AloeGuvnor,
+  have,
+  isFull,
+  PitcherPlant,
+  StealingMagnolia,
+} from "libram/dist/resources/2013/Florist";
```
Comment (horrible-little-slime) [inline src/post.ts:21]: import FloristFriar from libram instead

File: src/post.ts [hunk 2]
```diff
@@ -51,6 +59,11 @@ function coldMedicineCabinet(): void {
   }
 }
 
+function floristFriars(): void {
+  if (!have() || myLocation() !== $location`Barf Mountain` || isFull()) return;
+  [StealingMagnolia, AloeGuvnor, PitcherPlant].forEach((flower) => flower.plant());
```
Comment (horrible-little-slime) [inline src/post.ts:64]: make sure to have tested this; I have never personally used this code lol

Comment (MrFizzyBubbs) [reply src/post.ts]: Wasn't able to test, looks like libram florist friar code is broken

**Comment by horrible-little-slime** [comment]
Weirdly the merge conflicts don't look debilitating here; rebase it to main and test it and you've got yourself a done deal

**Comment by MrFizzyBubbs** [comment]
Tested successfully

**Pull request merged**